x-pack/legacy/plugins/task_manager/index.js

```diff
@@ -61,8 +63,10 @@ export function taskManager(kibana) {
         task: {
           hidden: true,
           isNamespaceAgnostic: true,
-          indexPattern: TASK_MANAGER_INDEX,
           convertToAliasScript: `ctx._id = ctx._source.type + ':' + ctx._id`,
+          indexPattern(config) {
+            return config.get('xpack.task_manager.index');
+          },
```

Comment:
Ah I was expecting the entire `savedObjectSchema` to be a function to be more similar to other uiExports. If you're confident there's no use case for that, then this is fine.

Reply:
Yeah I can't think of any yet. I believe if we make a function at the `savedObjectSchema` level and use reducers, I don't think the `config` object is available at that time. But so far this is the only configurable property I can think of.